### DISCUSSION TITLE
Escritas

### DISCUSSION THREAD

**Discussion by 123Will** [balta-io/forum#16] [Tira Dúvidas] [answered]
![image](https://user-images.githubusercontent.com/97917861/190276002-633324c8-bc15-4e90-8a4d-26ee8684ae4d.png)
 

Boa noite professor @andrebaltieri , Algumas escritas quando começa a digitar ela automaticamente já entra na tela, mas tem algumas que não entra, e nesse print mostra elas, e por esse motivo eu tive que pular algumas aulas, eu não consigo roda o código por que da erro.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @123Will, como vai?

Isto ocorre pois o `namespace` no qual o `CultureInfo` se encontra não foi importado.
As vezes o Visual Studio consegue encontrar o namespace e sugerir sua importação, mas nem sempre isto acontece.

A solução neste caso é importar manualmente o namespace:
```csharp
using System.Globalization;
```
